Cue/app/tabs/search/SearchHome.js

```diff
@@ -1,64 +1,113 @@
 // @flow
 
 import React from 'react'
-import { View, Text, Navigator, Platform } from 'react-native'
+
+import { searchDecks } from '../../actions'
+import type { State } from '../../reducers/user'
+import type { DeckMetadata } from '../../api/types'
+
+import { View, Text, Navigator, Platform, TextInput, Image  } from 'react-native'
 
 import { connect } from 'react-redux'
 
 import CueHeader from '../../common/CueHeader'
 import CueIcons from '../../common/CueIcons'
+import CueColors from '../../common/CueColors'
 
+import SearchListView from './SearchListView'
+
 const styles = {
   container: {
+    flex: 1,
+    backgroundColor: 'white'
+  },
+  bodyContainer: {
+    flex: 1,
+
+  },
+  searchBox: {
+    color: Platform.OS === 'android' ? 'white' : CueColors.primaryText,
+    backgroundColor: Platform.OS === 'android' ? 'transparent' : 'white',
+    borderRadius: 5,
     flex: 1,
-    backgroundColor: 'white'
+    fontSize: 16,
+    paddingLeft: 10,
+    paddingBottom: 6,
+    marginRight: Platform.OS === 'android' ? 90 : 16,
+    marginLeft: Platform.OS === 'android' ? 0 : 16,
+    height: Platform.OS === 'android' ? 50 : 35,
   },
```

Comment:
I dont think the text fits in the Input with padding 8? Or at least it doesn't when rendering the ios header in android